src/main/java/seedu/address/logic/commands/ShowCommand.java

```diff
@@ -145,6 +159,24 @@ private CommandResult showAssessment(Model model) throws CommandException {
         return new CommandResult(MESSAGE_SUCCESS, info, statistics.toHistogram());
     }
 
+    /**
+     * Executes command when a {@code Group} info is requested.
+     */
+    private CommandResult showGroup(Model model) throws CommandException {
+        assert getGroup().isPresent();
+        Group matchedGroup = model.getGroup(group);
+
+        if (matchedGroup == null) {
+            throw new CommandException(MESSAGE_NONEXISTENT_GROUP);
+        }
+
+        Info info = new Info(matchedGroup);
+        // uncomment when merging with graph display
+        // GroupStatistics statistics = new GroupStatistics(matchedGroup);
```

Comment:
pass in model as parameters
```suggestion
        GroupStatistics statistics = new GroupStatistics(matchedGroup, model);
```